dlt/cli/init_command.py

```diff
@@ -234,15 +243,30 @@ def init_command(
     destination_reference = Destination.from_reference(destination_type)
     destination_spec = destination_reference.spec
 
-    fmt.echo("Looking up the init scripts in %s..." % fmt.bold(repo_location))
-    clone_storage = git.get_fresh_repo_files(repo_location, get_dlt_repos_dir(), branch=branch)
-    # copy init files from here
-    init_storage = FileStorage(clone_storage.make_full_path(INIT_MODULE_NAME))
-    # copy dlt source files from here
-    sources_storage = FileStorage(clone_storage.make_full_path(SOURCES_MODULE_NAME))
-    # load init module and get init files and script
-    init_module = load_script_module(clone_storage.storage_path, INIT_MODULE_NAME)
+    # lookup core sources
+    local_path = Path(os.path.dirname(os.path.realpath(__file__))).parent / SOURCES_MODULE_NAME
```

Comment:
if explict repo location is present, disable the local sources. that allows to still use the old version or your own fork of verified sources

Reply:
I have added a flag for this now and amended some of the output to guide the user